src/java/org/apache/cassandra/service/paxos/PaxosPropose.java

```diff
@@ -126,6 +138,10 @@ private static class MaybeFailure extends Status
     final int participants;
     /** Number of accepts required */
     final int required;
+
+    /** Repairing an in flight txn not proposing a new one **/
+    final boolean isForRepair;
```

Comment:
could we call this `isForRecovery` to avoid confusion with actual repair?

Reply:
Renamed it